src/utils/textData.js

```diff
@@ -0,0 +1,757 @@
+const stopWords = [
```

Comment:
변하지 않는 상수들을 담은 배열이라 util 폴더가 아닌 constants 폴더를 따로 만들고, 변수 명은 대문자 스네이크 케이스를 적용 하는것은 어떨까요? 의견 여쭙습니다!
```suggestion
const STOP_WORDS = [
```

Reply:
서호님의 의견대로 상수로 처리하는 방향이 좋아보입니다!